packages/mdc-base/component.ts

```diff
@@ -21,55 +21,48 @@
  * THE SOFTWARE.
  */
 
-import MDCFoundation from './foundation';
+import MDCFoundation from "./foundation";
```

Comment:
We should absolutely not be using double quotes. AFAICT Google's style guides always promote single quotes for JS / TS.

The double quotes lint rule seems to be coming from [tslint:recommended](https://github.com/palantir/tslint/blob/master/src/configs/recommended.ts#L134). I'm surprised there's none in gts/tslint.json...

We should probably go with `'single'` to match our use.

Reply:
Ya, you're right. I'm surprised that gts doesn't have that. I will add that to our tslint file.